src/test/java/poll/repository/VoteRepositoryTest.java

```diff
@@ -1,4 +1,84 @@
 package poll.repository;
 
-public class VoteRepositoryTest {
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class VoteRepositoryTest {
+
+    private VoteRepository voteRepository;
+
+    @BeforeEach
+    void init() {
+        voteRepository = new VoteRepository();
+    }
+
+    @Test
+    void should_find_all_from_db() {
+        //given
+        //when
+        List<Vote> result = voteRepository.findAll();
+        //then
+        assertThat(result).hasSize(9);
+        assertThat(result.get(0).getName()).isEqualTo("Earvin Ngapeth");
```

Comment:
If you decided to check the first element, maybe it would be a good idea to check others as well. Also, as id field will be unique when we will have a database, I think I would check this field instead of the name. But in general you got the concept, so good job!